trestle/core/markdown/base_markdown_node.py

```diff
@@ -315,7 +280,7 @@ def _read_html_block(self, lines: List[str], line: str, i: int, ending_regex: st
                 break
         return html_block, i
 
-    def _read_table_block(self, lines: List[str], line: str, i: int) -> Tuple[str, int]:
+    def _read_table_block(self, lines: List[str], line: str, i: int) -> tuple[list[str], int]:
         """Read table."""
```

Comment:
I'm not clear how typing works here.  I thought using tuple rather than Tuple required Python 3.9 or later - but maybe it doesn't matter since it has no impact at run time.  I guess it only matters for people who are developing in 3.7 or 3.8?

Reply:
My understanding that it is not required but `typing` became deprecated in Python 3.9 (see here [PEP 585](https://peps.python.org/pep-0585/)). I think with the idea that we will drop the support of Python 3.7-3.8 we should start using `buildins` (i.e. `tuple`) rather than typing.